### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by martinbethann** [balloob/react-sidebar#88]
It would be lovely to see some accessibility improvements for this menu.  For example, being able to hit escape to close the menu and proper aria attributes for the open and closed state.

**Comment by Fallenstedt** [comment]
@martinbethann 
Aria attributes are indeed missing from this sidebar. I'll see what I can do to add aria-labels and roll attributes to a vanilla sidebar. I also like the idea of using the 'esc' as an access key for [keyboard accessibility](http://webaim.org/techniques/keyboard/accesskey)

**Comment by Fallenstedt** [comment]
By design, React Sidebar does not keep track of whether it is open or not. This has to be done by the parent component. This allows the parent component to make changes to the sidebar and main content based on the open/docked state. An example could be to hide the "show menu" button from the main content when the sidebar is docked. Or updating an aria attribute...

When a sidebar is used, I will leave it up to the user to create proper aria attributes for use.
